KsApi/models/graphql/adapters/GraphUserCreditCard+UserFragmentTests.swift

```diff
@@ -4,33 +4,42 @@ import XCTest
 
 class GraphUserCreditCard_UserFragmentTests: XCTestCase {
   func test() {
-    let userFragment = GraphAPI.UserFragment(
-      chosenCurrency: "USD",
-      email: "foo@bar.com",
-      hasPassword: true,
-      id: GraphQLID(),
-      imageUrl: "http://www.kickstarter.com/medium.jpg",
-      isAppleConnected: true,
-      isCreator: false,
-      isDeliverable: true,
-      isEmailVerified: true,
-      name: "Hari Singh",
-      storedCards: GraphAPI.UserFragment
-        .StoredCard(
-          nodes: [
-            GraphAPI.UserFragment.StoredCard
-              .Node(
-                expirationDate: "2023-01-01",
-                id: "123456",
-                lastFour: "4242",
-                type: .visa
-              )
-          ],
-          totalCount: 1
-        ),
-      uid: "12345"
-    )
+    let variables = ["withStoredCards": true]
+    let sampleCardDict: [String: Any] = [
+      "__typename": "UserCreditCardTypeConnection",
+      "nodes":
+        [[
+          "__typename": "CreditCard",
+          "expirationDate": "2025-02-01",
+          "id": "69021256",
+          "lastFour": "4242",
+          "type": "VISA"
+        ]],
+      "totalCount": 1
+    ]
 
-    XCTAssertNotNil(GraphUserCreditCard.graphUserCreditCard(from: userFragment))
+    do {
+      let storedCardsFragment = try GraphAPI.UserFragment
+        .StoredCard(jsonObject: sampleCardDict, variables: variables)
+      let userFragment = GraphAPI.UserFragment(
+        chosenCurrency: "USD",
+        email: "foo@bar.com",
+        hasPassword: true,
+        id: GraphQLID(),
```

Comment:
Found out yesterday that a standard String of an id works in place of the GraphQLID object. Can you try replacing that here and see if it works?